silverback/_cli.py

```diff
@@ -322,8 +322,9 @@ def list_clusters(platform: PlatformClient, workspace: str):
     if not (workspace_client := platform.workspaces.get(workspace)):
         raise click.BadOptionUsage("workspace", f"Unknown workspace '{workspace}'")
 
-    if cluster_names := list(workspace_client.clusters):
-        click.echo(yaml.safe_dump(cluster_names))
+    if clusters := workspace_client.clusters.values():
```

Comment:
Can you check to see if this works on a workspace with no clusters? I think we might have to wrap that with `list(...)`

Reply:
A workspace with no clusters will show 
```
$ silverback cluster list speed2

No clusters for this account
```